src/Templating/Helper/BlockHelper.php

```diff
@@ -487,12 +200,8 @@ public function render($block, array $options = []): string
             $stats['cache']['lifetime'] = $stats['cache']['age'] + $stats['cache']['ttl'];
```

Comment:
someone has an idea how useful those stopwatch traces/stats are now without the caching functionalities? 

Do we need the `cache` part of the trace?

```
 * @phpstan-type Trace = array{
 *     name: string,
 *     type: string,
 *     duration: int|float|false,
 *     memory_start: int|false,
 *     memory_end: int|false,
 *     memory_peak: int|false,
 *     cache: array{
 *         keys: mixed[],
 *         contextual_keys: mixed[],
 *         handler: false,
 *         from_cache: false,
 *         ttl: int,
 *         created_at: int|false|null,
 *         lifetime: int,
 *         age: int,
 *     },
 *     assets: array{
 *         js: string[],
 *         css: string[],
 *     }
 * }
 ```

Reply:
Maybe it should be rename 'response' since it's not related to cache anymore.
Dunno how usefull it is...